src/release_notes_workflow/component_release_notes.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+from typing import Any
+from manifests.input_manifest import InputComponentFromSource
+
+
+class ComponentReleaseNotes:
+
+    def __init__(self, component: InputComponentFromSource, build_version: str, root: str) -> None:
+        self.component = component
+        self.build_version = build_version
+        self.root = root
+        self.filename
```

Comment:
What’s this? did you mean to assign _filename?

Reply:
I have modified the logic to use property overloading.